projects/ng-components/lib/tooltip/tooltip-content/tooltip-content.component.html

```diff
@@ -1,15 +1,9 @@
-<div
-    #tooltip
-    @fadeOut
-    class="custom-tooltip-content tooltip bs-tooltip-{{ position }}"
-    [ngClass]="[theme, className]"
-    tabindex="-1"
-    role="tooltip"
-    (blur)="onBlur($event)"
->
-    <div class="arrow"></div>
-    <div class="tooltip-inner">
-        <ng-container *ngIf="isTemplateRef; else defaultContent" [ngTemplateOutlet]="content"></ng-container>
-        <ng-template #defaultContent>{{ content }}</ng-template>
+<div class="sebng-tooltip bs-tooltip-{{ position }}" [ngClass]="[theme, className]">
+    <div class="custom-tooltip-content">
+        <div class="tooltip-inner" id="aria-label" role="tooltip">
```

Comment:
why is the `id` set to `aria-label`??

Reply:
To associate one element with another as its primary label, you I used aria-labelledby. The relationship is forged by the aria-labelledby and id attributes sharing the same value.

in tooltip.component.html:
```html 
<span aria-labelledby="aria-label"></span>
```